docs/en/latest/discovery/nacos.md

```diff
@@ -60,7 +99,7 @@ discovery:
 
 #### L7
 
-Here is an example of routing a request with an URI of "/nacos/*" to a service which named "http://192.168.33.1:8848/nacos/v1/ns/instance/list?serviceName=APISIX-NACOS" and use nacos discovery client in the registry:
+Here is an example of routing a request with an URI of "/nacos/*" to a service which named "http://192.168.33.1:8848/nacos/v1/ns/instance/list?serviceName=APISIX-NACOS" configured with id `nacos-cluster-1` and use nacos discovery client in the registry:
 
 :::note
 You can fetch the `admin_key` from `config.yaml` and save to an environment variable with the following command:
```

Comment:
Not the right place to have this note under L7. Move it one level up.

Reply:
Do you mean the :::note section? It was here already. Do you want me to move it?